core/src/main/scala/org/apache/spark/internal/config/History.scala

```diff
@@ -125,4 +125,18 @@ private[spark] object History {
   val KERBEROS_KEYTAB = ConfigBuilder("spark.history.kerberos.keytab")
     .stringConf
     .createOptional
+
+  val CUSTOM_EXECUTOR_LOG_URL = ConfigBuilder("spark.history.custom.executor.log.url")
+    .doc("Specifies custom spark executor log url for supporting external log service instead of " +
+      "using resource managers' app log urls. Spark will support some path variables via " +
+      "patterns which can be vary on resource managers. Please check the document for " +
+      "resource manager to refer which patterns are supported.")
```

Comment:
Handful of small corrections:

```
"Specifies custom spark executor log url for supporting external log service instead of " +
"using cluster managers' application log urls in the history server. Spark will support some path variables via " +
"patterns which can vary on cluster manager. Please check the documentation for " +
"your each cluster manager to see which patterns are supported, if any.  " +
"This configuration has no effect on a live application, it only affects the history server."
```